Code/GraphMol/AddHs.cpp

```diff
@@ -508,6 +508,39 @@ void removeHs(RWMol &mol, bool implicitOnly, bool updateExplicitCount,
           heavyAtom->setProp(common_properties::_UnknownStereo, 1);
         }
 
+        // if the direction is set on this bond and the atom it's connected to
+        // has no other single bonds with directions set, then we need to set
+        // something in order to avoid double bond stereochemistry possibly
```

Comment:
Perhaps a bit more specific "do something" -> flip the stereochemistry from the hydrogen to the heavy atom, i.e.  =C(/[H])C => =C\C